### ISSUE TITLE
Set of questions we would ask and why - to form basis of workflow later

### ISSUE THREAD

**Issue by jim-smith** [AI-SDC/SACRO-ML#29]

**Comment by jim-smith** [comment]
| Question | Purpose | Comments |
|--------|--------|--------|
| Have they provided a 'target' object | Means we can run all the tests we want to have more confidence in recommendations | Cell |
| If  transfer learning has been used have they fine tuned the whole network or just trained the final layers | to determine the number of degrees of freedom in the model | obviously less chance of 'remembering data if they have not fine-tuned the layers that provide the 'detour creation/embedding |
| Can multiple records relate to the same person? (e.g. repeat hospital visits, multiple scans, ...| To determine whether DP is appropriate / how to adjust \epsilon values | DP guarantees rise as power of number of records relating to the same person |
| Have they used an instance-based classifier | Highly likely that the model contains training data | DP-variants are available |

**Comment by drchriscole** [comment]
Questions we at HIC are asking:

- have input data been pre-processed and how? (if averaging methods have been implemented then an MIA will at worst reveal an average "individual")
- when fine tuning what is the size of the model pre/post tuning? (a significant increase in model size may indicate inclusion of data)
- what is the performance of the model on training data and a unseen validation set? (may help the decision process with borderline results. In future the TRE would provide the validation set).
